### PR TITLE
Add videos and projects

### PR DESCRIPTION
**Pull request by myagues**
I have added some video presentations and models built with JAX/Flax. There is also a recent tutorial, half in JAX and PyTorch, not sure whether to put it as a video or project, or if it is out of the scope of the list:

- [Deep Implicit Layers - Neural ODEs, Deep Equilibirum Models, and Beyond | NeurIPS 2020](https://slideslive.com/38935810/deep-implicit-layers-neural-odes-equilibrium-models-and-beyond) - tutorial created by Zico Kolter, David Duvenaud, and Matt Johnson with Colab notebooks avaliable in [Deep Implicit Layers](http://implicit-layers-tutorial.org).

Also some interesting blog posts from Eric Jang: [Differentiable Path Tracing](https://blog.evjang.com/2019/11/jaxpt.html), [Normalizing Flows](https://blog.evjang.com/2019/07/nf-jax.html), [Meta-Learning](https://blog.evjang.com/2019/02/maml-jax.html) and Matt Kelcey: [Ensemble Nets](http://matpalm.com/blog/ensemble_nets) and [out of distribution (OOD) detection](http://matpalm.com/blog/ood_using_focal_loss), but it is difficult to set a bar for which blog posts should or should not be added.

Also, Ross Wightman is building a repo with some pretrained models, [efficientnet-jax](https://github.com/rwightman/efficientnet-jax), with both Objax and Flax, although it is still a work in progress.

### CODE REVIEW

**Comment by n2cholas** [comment]
Thanks for the PR @myagues! Especially for the models contributions--I had a hard time finding stuff for that. 

Could you add the papers you listed (Fourier Feature Networks and Vision Transformer) to the papers section as well? I realize a lot of the projects/models will be duplicated in the Papers section, and I'm not sure what the best way to deal with that will be. Any thoughts? Perhaps we can remove the papers section altogether and only link relevant implementations?

I think the video you posted is relevant, I quickly skimmed it and the implicit layers implementation is very insightful for JAX users, so please add that to the videos list.

I think the blog posts you listed are nice and should be in our blog posts section. I don't really have a bar yet. My initial plan was to let the list grow organically, then eventually remove posts that have duplicated content (and keep the one that is higher quality). If you have a better recommendation, I am open to hearing it.

**Comment by myagues** [comment]
> Any thoughts? Perhaps we can remove the papers section altogether and only link relevant implementations?

I think both sections can be of use. I think the paper section can have a brief description + authors and arXiv or openreview link, so people can get a grasp where JAX is being used and in which domains, whereas the model section is more of a collection of current implementations + old ports. Also, we can link to model webpages there instead of adding more links to the paper section.

To bridge the papers with its own implementation, I have added a handle to each model and a small mark to the corresponding paper that links to the code. The downside is the bullets in the Models section are more spaced-out than in the others.

> My initial plan was to let the list grow organically, then eventually remove posts that have duplicated content (and keep the one that is higher quality).

I have added the blog posts. I agree, it is better to have them so people can have more sources of information to learn from.

**Comment by n2cholas** [comment]
Thanks @myagues, the PR looks good, I'll merge it in.
